libs/store/src/lib/infrastructure/persistence/store/entity-store-builder.spec.ts

```diff
@@ -31,25 +31,16 @@ class EntityCollectionServiceFactoryMock implements EntityCollectionServiceFacto
     }
 }
 
```

Comment:
I cannot comment above at the header of the file so just adding here. 

import { QueryAdapterFactory } from '../query/query-adapter'; unused import

Reply:
Removed unused imports